examples/utils/migration-tools/src/migrationTool.ts

```diff
@@ -65,14 +70,15 @@ class MigrationTool implements IMigrationTool {
 			return "migrated";
 		} else if (this.acceptedMigration !== undefined) {
 			return "migrating";
+			// eslint-disable-next-line unicorn/no-negated-condition
```

Comment:
Yeah, I'm usually not a fan but will go along with it.  Here though I decided to at least put off compliance since it would want me to swap just the last two clauses that are otherwise in chronological order.